microservices/workers/src/Worker/Xmlrpc.php

```diff
@@ -5,6 +5,7 @@
 use Doctrine\ORM\EntityManagerInterface;
```

Comment:
This file is called _Xmlrpc.php_ but does not longer use XML at all, doesn't it?

Reply:
You're right. We'll rename them (jobs, etc.) in another PR